src/dataHelpers.jl

```diff
@@ -19,8 +19,8 @@ const murineFcgR = [:FcgRI, :FcgRIIB, :FcgRIII, :FcgRIV]
 const humanFcgR =
     Symbol.(["FcgRI", "FcgRIIA-131H", "FcgRIIA-131R", "FcgRIIB-232I", "FcgRIIB-232T", "FcgRIIC-13N", "FcgRIIIA-158F", "FcgRIIIA-158V", "FcgRIIIB"])
 const murineActI = [1, -1, 1, 1]
-const murineResponse = [8e4, 3.5e4, 2.5e-1, 7e3, 3]
-const humanResponse = [6e4, 2e5, 4.5e4, 3.5e4, 3e3]
+const murineResponse = [8e4, 5e3, 2.5e-1, 7e3, 3]
```

Comment:
What are these vectors?

Reply:
@aarmey Those are the ymax for each cell activity in the S plots. I could move them within the figure building function if that would make more sense?

Reply:
Ok. Please add a comment noting this.